semgrep_output_v1.atd

```diff
@@ -2514,6 +2514,13 @@ type transitive_reachability_filter_params = {
   findings: transitive_finding list;
   dependencies: resolved_dependency list;
 }
+(* ----------------------------- *)
+(* Symbol analysis upload response*)
+(* ----------------------------- *)
+
+type symbol_analysis_upload_response = {
```

Comment:
I'd make it a `uri` instead of a `string`, which communicates its purpose at the type level. Otherwise looks good. Maybe add a comment explaining that this is a presigned aws url?